deploy/kubernetes/base/controller/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
+- controller.yaml
+
+commonLabels:
```

Comment:
what is this doing? why do we need to kustomize this, shouldn't all deployments have this label?